features/gezag-minderjarige/1.4-is-uitspraak-gezag-aanwezig.feature

```diff
@@ -0,0 +1,150 @@
+      Gegeven voor 'Babette' is een gerechtelijke uitspraak over het gezag gedaan met de volgende gegevens
+      | naam                                 | waarde                          |
+      | indicatie gezag minderjarige (32.10) | 1                               |
+      | ingangsdatum geldigheid (85.10)      | 20230101                        |
+      | aanduiding in onderzoek (83.10)      | 113210                          |
+      | datum ingang onderzoek (83.20)       | 20230101                        |
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                                                                                                                                      |
+      | type                             | GezagNietTeBepalen                                                                                                                          |
+      | toelichting                      | Gezag is niet te bepalen, omdat de volgende relevante gegevens in onderzoek staan. Persoonslijst van persoon: indicatie gezag minderjarige. |
+
+    Scenario: Voor het kind een uitspraak gedaan, deze uitspraak stond in onderzoek er is sprake van EenhoofdigOuderlijkGezag
+      Gegeven voor 'Babette' is een gerechtelijke uitspraak over het gezag gedaan met de volgende gegevens
+      | naam                                 | waarde                          |
+      | indicatie gezag minderjarige (32.10) | 1                               |
+      | ingangsdatum geldigheid (85.10)      | 20230101                        |
+      | aanduiding in onderzoek (83.10)      | 113210                          |
+      | datum ingang onderzoek (83.20)       | 20230101                        |
+      | datum einde onderzoek (83.30)        | 20231001                        |
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                       |
+      | type                             | EenhoofdigOuderlijkGezag     |
+      | minderjarige.burgerservicenummer | 000000036                    |
+      | ouder.burgerservicenummer        | 000000012                    |
+
+  Regel: Uitspraak gezag is aanwezig
```

Comment:
Voor deze regel is alleen een scenario geschreven voor gezagsindicatie "2". Vanuit test perspectief zou je elke gezagsindicatie af moeten gaan: "1", "2", "12", "1D", "2D", "D".